docs/4.0/examples/sticky-footer/index.html

```diff
@@ -19,13 +19,13 @@
   <body>
 
     <!-- Begin page content -->
-    <div class="container">
+    <main role="container" class="container">
```

Comment:
Typo here.

Reply:
Good spot!

Reply:
vnu.jar in action, which is why we need to bring the HTML validator back. :)

Reply:
Fixed

Reply:
Don't forget to add the PR in the ship notes.